tests/test_ase.py

```diff
@@ -0,0 +1,70 @@
+# TODO: test for rotational invariance, translation invariance,
+# and permutation, as well as size extensivity
+import numpy as np
+import pytest
+from ase.build import bulk
+
+from shiftml.ase import ShiftML
+
+expected_output = np.array([137.5415, 137.5415])
+
+
+def test_shiftml1_regression():
+    """Regression test for the ShiftML1.0 model."""
+
+    frame = bulk("C", "diamond", a=3.566)
+    model = ShiftML("ShiftML1.0", force_download=True)
```

Comment:
why `force_download=True`?

Reply:
This was to make sure that the local tests do not resuse a local potentially old cached file.